src/hooks/api/useMonthlyWorth.ts

```diff
@@ -33,31 +33,29 @@ export function useMonthlyWorth(
   const { data: accounts, dataUpdatedAt: accountsUpdatedAt } = useAccounts();
   const { data: prices, dataUpdatedAt: pricesUpdatedAt } = usePrices({});
   const { data: totals, dataUpdatedAt: totalsUpdatedAt } = useBalanceSheet(
-    interval.start?.endOf('month') as DateTime<true>,
+    (interval.start?.minus({ day: 1 }) as DateTime),
     (data) => data,
   );
 
   const aggregate = React.useCallback(
     ((data: AccountsTotals[]) => {
-      const dates = monthlyDates(interval as Interval).map(d => d.endOf('month'));
-      dates[dates.length - 1] = (interval as Interval).end as DateTime;
       const aggregated = aggregateMonthlyWorth(
         ['type_asset', 'type_liability'],
         accounts as Account[],
         [
           totals as AccountsTotals,
           ...data,
         ],
-        dates,
       );
 
+      const dates = [interval.start as DateTime, ...intervalToDates(interval as Interval)];
       return aggregated.map((d, i) => aggregateChildrenTotals(
         ['type_asset', 'type_liability'],
         accounts as Account[],
         prices as PriceDBMap,
         dates[i],
         d,
-      ));
+      )).slice(1);
```

Comment:
we now compute the previous total until the start of the interval and then add it to the monthly worth. Once added, we can remove the first array as otherwise it creates a series with 1 extra data point